src/responses/map.ts

```diff
@@ -153,6 +158,124 @@ export async function StatesLegendMapResponse() {
     .toBuffer();
 }
 
+// Begin history map respones
+export async function DistrictsHistoryMapResponse(
+  mapType: string, // "map" for map without legend, "legendMap" for map with legend
```

Comment:
Let's use an `enum` for `mapType` and not a `string`

Reply:
Let's also add a default value for `mapType`. This ensure the response always returns something.